website_and_docs/content/documentation/selenium_manager.en.md

```diff
@@ -118,7 +132,7 @@ Selenium Manager includes two additional arguments two handle the cache, namely:
 Selenium Manager follows the same versioning schema as Selenium. Nevertheless, we use the major version 0 for Selenium Manager releases because it is still in beta. For example, the Selenium Manager binaries shipped with Selenium 4.12.0 corresponds to version 0.4.12.
 
 ## Getting Selenium Manager
-For most users, Selenium Manager should work silently and transparently. But if you want to *play* with Selenium Manager or use it for your use case, you can get the Selenium Manager binaries in different ways:
+For most users, direct interaction with Selenium Manager is not required since the Selenium bindings use it internally. Nevertheless, if you want to *play* with Selenium Manager or use it for your use case involving driver or browser management, you can get the Selenium Manager binaries in different ways:
 
 - From the cache. As of Selenium 4.12.0, the Selenium Manager binaries are extracted from each binding distribution and copied to the cache folder. For instance, the Selenium Manager binary shipped with Selenium 4.12.0 will be stored in the folder `~/.cache/selenium/manager/0.4.12`).
```

Comment:
This isn't going to be supported in 4.12. Too much more to add. Let's say that it's under development.

Reply:
That's a pity. Will change.

Reply:
Changed.